ansible_collections/arista/avd/roles/eos_designs/schemas/schema_fragments/defs_node_type.schema.yml

```diff
@@ -328,6 +328,24 @@ $defs:
               Note! The content of this dictionary is _not_ validated by the schema, since it can be either ethernet_interfaces or port_channel_interfaces.
             # Schema ref not implemented since we do not know which of the schemas to look at.
             # $ref: "eos_cli_config_gen#/keys/ethernet_interfaces/items/"
+          downlink_pools:
+            documentation_options:
+              table: node-type-uplink-configuration
+            description: IPv4 subnets to use to connect to downlink switches. Set this on the parent switch. Cannot be combined with uplink_ipv4_pool for the same link.
+            type: list
+            items:
+              type: dict
+              keys:
+                downlink_ipv4_pool:
+                  type: str
+                  format: ipv4_cidr
+                  description: IPv4 subnet to use to connect to downlink switches
+                downlink_interfaces:
```

Comment:
Not sure on this.
```suggestion
                interfaces:
```

Reply:
I asked the team about the names and they said they don't really mind, and I feel the same way personally. I think you would be a better judge on this @ClausHolbechArista.

Reply:
@carlbuchmann any preferences here?

Reply:
I think the `downlink_` prefixes are not necessary.